src/main/scala/ai/privado/entrypoint/ScanProcessor.scala

```diff
@@ -442,7 +442,7 @@ object ScanProcessor extends CommandProcessor {
           appCache,
           propertyFilterCache = propertyFilterCache,
           statsRecorder = statsRecorder
-        ).createPythonCpg()
+        ).processCpg()
```

Comment:
databaseDetailsCache also need to be passed

Reply:
done.